be/src/common/config.h

```diff
@@ -59,6 +59,13 @@ CONF_String(ssl_private_key_path, "");
 // These connections are created during the first few access and will be used thereafter
 CONF_Int32(brpc_max_connections_per_server, "1");
 
+// BRPC stub cache cleanup configurations
+// Interval in seconds to run cleanup for expired BRPC stubs
+CONF_mInt32(brpc_stub_cleanup_interval_s, "600"); // 10 minutes
```

Comment:
would you please also add the new configurations to `docs/en/administration/management/BE_configuration.md`